### PR TITLE
feat(avatars): enable default avatar

### LINKED ISSUE #295: Provide a default avatar URL if the user hasn't uploaded one?
From discussion in https://github.com/mozilla-lockbox/lockbox-extension/issues/417

We have a nice "placeholder" avatar image that we use in our UI when the user hasn't uploaded a custom image.  For consistency it would be nice for reliers to be able to use the same image.

For users who have not set an avatar image, should we just return the URL of the placeholder image as part of the user's profile data?

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #295

### CODE REVIEW

**Comment by vladikoff** [comment]
@rfk r?

Added information into the deploy doc after talking this through with @jrgm

File: docs/API.md [hunk 1]
```diff
@@ -166,12 +166,36 @@ curl -v \
 "https://profile.accounts.firefox.com/v1/avatar"
 ```
 
-#### Response
+
+#### Response 
+
+> Note: we no longer support Gravatar uploading, but there are entries in the database
```
Comment (rfk) [inline docs/API.md]: Are there?  Side note, but I wonder if we should somehow purge them.

File: lib/server/_static.js [hunk 2]
```diff
@@ -20,6 +28,26 @@ exports.create = function() {
     port: config.server.port + 1
   });
 
+  server.route({
+    method: 'GET',
+    path: '/a/' +  DEFAULT_AVATAR_ID + '{type?}',
```
Comment (rfk) [inline lib/server/_static.js]: 👍  I like this way of just making it all a bit simpler for dev

File: docs/API.md (hunk 1 above)
Comment (vladikoff) [reply docs/API.md]: Mistake, we actually used to process them, I guess the docs always had an incorrect url

**Pull request merged**